test/fixtures/settings.yml

```diff
@@ -404,3 +404,28 @@ attribute87:
   category: Setting::General
   default: '2abbbe02-4ace-4269-9e20-2753f3206cc2'
   description: 'Foreman UUID'
+attribute88:
+  name: oidc_jwks_url
+  category: Setting::Auth
+  default: 127.0.0.1
```

Comment:
This default is very different from the description since it's not a URL. If there is no sane default, should it be empty?